lookout/style/format/rules.py

```diff
@@ -49,18 +49,18 @@ class Rules:
 
     _log = logging.getLogger("Rules")
 
-    def __init__(self, rules: List[Rule], origin: Mapping[str, Any]):
+    def __init__(self, rules: List[Rule], config: Mapping[str, Any]):
```

Comment:
`origin` is more expressive here because we do not parameterize anything here, just include the origin-al configuration.
It's like naming a counter "integer": yep, it's an integer but no use of that name.

Reply:
can I use `origin_config`? because IMHO `origin` also not understandable.